Modules/FindLCContent.cmake

```diff
@@ -0,0 +1,53 @@
+### MIGRATE-NO-ACTION
```

Comment:
I think this find module can be removed - LCContent is the linear collider equivalent to LArContent, so LArContent should never have a need to find this package.